src/backends/backend/triton_backend_manager.cc

```diff
@@ -356,7 +360,19 @@ TritonBackendManager::CreateBackend(
       name, dir, libpath, backend_cmdline_config, backend));
   singleton_manager.backend_map_.insert({libpath, *backend});
 
+  const char* backend_config;
```

Comment:
What about when a backend unloads? Don't try to update the BackedState incrementally, just calculate it each time BackendState() is called.